src/main/scala/lms/thirdparty/cuda/cuda.scala

```diff
@@ -661,10 +661,12 @@ trait CudaOps extends Dsl with StackArrayOps with SizeTOps with CLibs with CudaF
 
 
   // Some global values for our GPU
+  val tileDim = 32
```

Comment:
Nit: I probably want to move the declaration of `tileDim` and `blockRows` right before the `cudaTranspose` function, since they are not global values for GPU, but rather variables used by the transpose kernel